script/main.lua

```diff
@@ -35,1925 +35,1994 @@ rtb = {}
+    c.sendChannel(chan, con, "LRP", { channel = c.getName(chan), character = u.getName(con), message = s.escapeHTML(args.message) })
+    return const.FERR_OK
 end
 
 -- Sends a chat message to a channel.
 -- Syntax: MSG <channel> <message>
 event.MSG =
-function (con, args)
-	if args.channel == nil or args.message == nil then
-		return const.FERR_BAD_SYNTAX
-	end
-
-	if u.checkUpdateTimer(con, "msg", const.MSG_FLOOD) == true then
-		return const.FERR_THROTTLE_MESSAGE
-	end
-
-	local found, chan = c.getChannel(string.lower(args.channel))
-	if found ~= true then
-		return const.FERR_CHANNEL_NOT_FOUND
-	end
-
-	if c.getMode(chan) == "ads" then
-		return const.FERR_ADS_ONLY
-	end
-
-	local chantype = c.getType(chan)
-	if (chantype == "private" or chantype == "pubprivate") and #args.message > const.PRI_MAX then
-		return const.FERR_MESSAGE_TOO_LONG
-	elseif #args.message > const.MSG_MAX then
-		return const.FERR_MESSAGE_TOO_LONG
-	end
-
-	if c.inChannel(chan, con) ~= true then
-		return const.FERR_NOT_IN_CHANNEL
-	end
-	
-	if hasShortener(args.message) ~= false then
-                return const.FERR_OK
-        end
-
-	if u.getMiscData(con, "hellban") ~= nil then
-		return const.FERR_OK
-	end
-
-	-- s.logMessage("message", con, chan, nil, args.message)
-	c.sendChannel(chan, con, "MSG", {channel=c.getName(chan), character=u.getName(con), message=s.escapeHTML(args.message)})
-	return const.FERR_OK
+function(con, args)
+    if args.channel == nil or args.message == nil then
+        return const.FERR_BAD_SYNTAX
+    end
+
+    if u.checkUpdateTimer(con, "msg", const.MSG_FLOOD) == true then
+        return const.FERR_THROTTLE_MESSAGE
+    end
+
+    local found, chan = c.getChannel(string.lower(args.channel))
+    if found ~= true then
+        return const.FERR_CHANNEL_NOT_FOUND
+    end
+
+    if c.getMode(chan) == "ads" then
+        return const.FERR_ADS_ONLY
+    end
+
+    local chantype = c.getType(chan)
+    if (chantype == "private" or chantype == "pubprivate") and #args.message > const.PRI_MAX then
+        return const.FERR_MESSAGE_TOO_LONG
+    elseif #args.message > const.MSG_MAX then
+        return const.FERR_MESSAGE_TOO_LONG
+    end
+
+    if c.inChannel(chan, con) ~= true then
+        return const.FERR_NOT_IN_CHANNEL
+    end
+
+    if hasShortener(args.message) ~= false then
+        return const.FERR_OK
+    end
+
+    if u.getMiscData(con, "hellban") ~= nil then
+        return const.FERR_OK
+    end
+
+    s.logMessage("message", con, chan, nil, args.message)
+    c.sendChannel(chan, con, "MSG", { channel = c.getName(chan), character = u.getName(con), message = s.escapeHTML(args.message) })
+    return const.FERR_OK
 end
 
 -- Gets the list of open private channels.
 -- Syntax: ORS
 event.ORS =
-function (con, args)
-	if u.checkUpdateTimer(con, "ors", const.ORS_FLOOD) == true then
-		return const.FERR_OK
-	end
-	c.sendORS(con)
-	return const.FERR_OK
+function(con, args)
+    if u.checkUpdateTimer(con, "
```

Comment:
This seems like it wouldn't belong here, but in getModList. Here it just overrides the check of whether the list is empty.